src/quo2/components/share/share_qr_code/component_spec.cljs

```diff
@@ -0,0 +1,33 @@
+(ns quo2.components.share.share-qr-code.component-spec
+  (:require [quo2.components.share.share-qr-code.view :as share-qr-code]
+            [test-helpers.component :as h]))
+
+(h/describe "Profile Input"
```

Comment:
I believe the `Profile Input` string needs to change to the correct component name.

Reply:
whoops, good spot :D